Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -77,8 +85,7 @@ def get_triage_events_by_num(self, event):
         """
             gets triage events by number
         """
-        event = event - 1
         triage_url = self.url + "triage-item-events?limit=20&event-num-after=" + str(event)
         response = requests.get(triage_url, headers={"Authorization": "Basic %s" % self.b64val, "searchlight-account-id": "%s" % self.id})
-        logging.info("Events by num response code: %s" % response.status_code)
+        logging.info("Events by num %d response code: %s" % (event, response.status_code))
```

Comment:
This thing actually wont matter because in Sentinel it doesn't show it like `DS_api - my log message` neither on local run or on Azure portal. It still shows it like earlier somehow.